documentation/src/main/asciidoc/topics/proc_using_mvn_archetypes.adoc

```diff
@@ -0,0 +1,58 @@
+[id='mvn_archetypes']
+= Creating {brandname} Projects
+{brandname} provides a set of Maven archetypes that you can use to generate
```

Comment:
would it make sense to add a link to https://maven.apache.org/guides/introduction/introduction-to-archetypes.html? for users that aren't aware of what it is.

Reply:
yeah, that'd be good. obviously I could do with reading that too...